nilearn/plotting/displays.py

```diff
@@ -1897,17 +1897,17 @@ def find_cut_coords(self, img=None, threshold=None, cut_coords=None):
         if (not isinstance(cut_coords, collections.abc.Sequence) and
                 isinstance(cut_coords, numbers.Number)):
             cut_coords = [cut_coords] * 3
-            cut_coords = self._find_cut_coords(img, cut_coords,
-                                               self._cut_displayed)
+            cut_coords = cls._find_cut_coords(img, cut_coords,
```

Comment:
thanks! for this to work we still need to make `_find_cut_coords` a static method

Reply:
Yes, see previous PR #2921.

Reply:
We do need to merge #2921 first.

Reply:
Actually, this works (not) the same way before/after the PR. The real fix is in #2921, this here is just a variable name change.